testsuite/tests/tool-dumpobj/test.run

```diff
@@ -0,0 +1,4 @@
+set -e -o pipefail
+
+# We remove the first and last lines as they contain user-dependent file paths
+${ocamlsrcdir}/tools/dumpobj.opt ${program} | tail -n +2 | head -n -1 > ${output}
```

Comment:
For the header part, you can ask ocamltest to skip header lines with the `skip_header_lines` variable:

```
**** check-program-output
skip_header_lines=1
```
It would probably make sense to update ocamltest with a `skip_footer_lines` variable for the symmetric part.